### DISCUSSION TITLE
Escaping double curly braces

### DISCUSSION THREAD

**Discussion by mstave** [artilleryio/artillery#1450] [Artillery]
I'm not trying to do variable substitution here, I'm trying to pass a string that has double curly braces 
```json
"qs" : {
     "someparam" : "foo{{ bar }}baz"
}
```
Artillery passes `fooundefinedbaz`

I tried `"foo{{{{ bar }}}}baz"` - it passes 4 of each brace.  `\{` isn't valid json.

I tried also putting the value in a variable and then referring to the variable, still `fooundefinedbaz`...
Any ideas?

**Comment by hassy** [comment]
hi @mstave 👋 Can't be done right now (an oversight in the templating code in Artillery). converting this into an Issue

**Reply by hassy** [reply]
tracking it here: https://github.com/artilleryio/artillery/issues/1456
